.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: C
+dist: xenial
```

Comment:
well, it isnt ... but it is also ignored and describes that we now use a xenial chroot inside when testing ;)